server/chat/backend/agent/tools/opensearch_tool.py

```
@@ -0,0 +1,158 @@
+            name = idx.get("index", "?")
+            health = idx.get("health", "?")
+            docs = idx.get("docs.count", "?")
+            size = idx.get("store.size", "?")
+            lines.append(f"  - {name} | health={health} | docs={docs} | size={size}")
+
+        if len(indices) > 100:
+            lines.append(f"  ... and {len(indices) - 100} more")
+
+        return "\n".join(lines)
+
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)